sidekick_core/lib/src/commands/deps_command.dart

```diff
@@ -47,14 +47,11 @@ class DepsCommand extends Command {
     final errorBuffer = StringBuffer();
 
     final excluded = excludeGlob
-        .map((p) => Glob(p))
-        .map(
-          (e) => e.listSync(
-            // See https://github.com/dart-lang/glob/issues/52
-            root: Directory.current.path,
-          ),
-        )
-        .expand((e) => e) // flattens nested list
+        .expand((rule) {
+          // start search at repo root
+          final root = repository.root.path;
+          return Glob("$root/$rule").listSync(root: root);
```

Comment:
does `/` work on Windows as well or do we have to use `Platform.pathSeparator` here as well?

Reply:
Yes, the [glob readme](https://pub.dev/packages/glob#syntax) states

> In order to be as cross-platform and as close to the Bash syntax as possible, all globs use POSIX path syntax, including using `/` as a directory separator regardless of which platform they're on. This is true even for Windows roots; for example, a glob matching all files in the C drive would be `C:/*`.